engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala

```diff
@@ -7,16 +7,16 @@ import pl.touk.nussknacker.engine.kafka.KafkaConfig
 
 import scala.collection.concurrent.TrieMap
 
-class AvroUtils(schemaRegistryClientFactory: SchemaRegistryClientFactory, kafkaConfig: KafkaConfig) extends Serializable {
-
-  private lazy val schemaRegistryClient = schemaRegistryClientFactory.createSchemaRegistryClient(kafkaConfig)
+class AvroUtils[T](schemaRegistryClientFactory: SchemaRegistryClientFactory[T], kafkaConfig: KafkaConfig) {
 
   private lazy val parsedSchemaCache = TrieMap.empty[String, Schema]
 
   private lazy val schemaBySubjectAndVersionCache = TrieMap.empty[(String, Int), Schema]
 
   private lazy val lastestSchemaBySubjectCache = TrieMap.empty[String, Schema]
 
+  private lazy val schemaRegistryClient = schemaRegistryClientFactory.createSchemaRegistryClient(kafkaConfig)
```

Comment:
Why this reordering is better than before?

Reply:
I didn't do it specially.. It was added after few changes.. I will move it to old place..